control-plane/subcommand/webhook-cert-manager/command.go

```diff
@@ -3,7 +3,6 @@ package webhookcertmanager
 import (
```

Comment:
Update webhook-cert-manager to use new comon code in `mutatingwebhookconfiguration.UpdateWithCABundle()`